plugins/ks.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ks
+spec:
+  version: v0.0.10
+  homepage: https://github.com/linuxsuren/ks
+  shortDescription: This plugin simplifies the work when you're managing or developing with KubeSphere.
+  description: |
+    This plugin helps you to manage the components of KubeSphere without typing a lot of words. 
+    Make it be much easier if you want to enable, disable, check the logs output, or even upgrade it.
```

Comment:
I think there is some language barrier. What do you think about the following wording? This should be easier to understand: 

> This plugin simplifies management of KubeSphere components. KubeSphere helps to run continuous delivery pipelines on Kubernetes. You can enable/disable, inspect logs, or upgrade KubeSphere components.